### PR TITLE
[MRG] Disable interactivity when topography information is not available.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/4370

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -297,6 +300,14 @@ def _plot_lines(data, info, picks, fig, axes, spatial_colors, unit, units,
                                            alpha=0.75)]
     gfp_path_effects = [patheffects.withStroke(linewidth=5, foreground="w",
                                                alpha=0.75)]
+    if selectable:
+        selectables = np.ones(len(ch_types_used), dtype=bool)
+        for type_idx, this_type in enumerate(ch_types_used):
+            idx = picks[types == this_type]
+            if len(idx) < 2 or (this_type == 'grad' and len(idx) < 4):
+                warn('No topography available for %s. '
```
Comment (mmagnuski) [inline mne/viz/evoked.py]: maybe something like "channel position" instead of topography?

**Comment by agramfort** [comment]
thanks @jaeilepp !

**Comment by cbrnr** [comment]
Does this really solve the problem? I can still drag an area in the PSD plot using EEG data with no topographic information (only channel names). First, I get this warning:

    WARNING:root:Did not find any electrode locations the info, will attempt to use digitization points 
    instead. However, if digitization points do not correspond to the EEG electrodes, this will lead to 
    bad results. Please verify that the sensor locations in the plot are accurate.

Which then results in the following error:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/matplotlib/backends/backend_qt5.py", line 313, in mouseReleaseEvent
    guiEvent=event)
  File "/usr/local/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 1940, in button_release_event
    self.callbacks.process(s, event)
  File "/usr/local/lib/python3.6/site-packages/matplotlib/cbook.py", line 554, in process
    proxy(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/matplotlib/cbook.py", line 416, in __call__
    return mtd(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/matplotlib/widgets.py", line 1292, in release
    self._release(event)
  File "/usr/local/lib/python3.6/site-packages/matplotlib/widgets.py", line 1536, in _release
    self.onselect(vmin, vmax)
  File "/Users/clemens/Repositories/mnelab/mne/viz/evoked.py", line 112, in _line_plot_onselect
    info, ch_type, layout=None)
  File "/Users/clemens/Repositories/mnelab/mne/viz/topomap.py", line 70, in _prepare_topo_plot
    pos = _find_topomap_coords(info, picks)
  File "/Users/clemens/Repositories/mnelab/mne/channels/layout.py", line 643, in _find_topomap_coords
    pos = _auto_topomap_coords(info, picks)
  File "/Users/clemens/Repositories/mnelab/mne/channels/layout.py", line 701, in _auto_topomap_coords
    raise RuntimeError('No digitization points found.')
RuntimeError: No digitization points found.
Abort trap: 6
```

**Comment by cbrnr** [comment]
@jaeilepp @agramfort @mmagnuski can you confirm if this is still an issue?

**Comment by jaeilepp** [comment]
I can't replicate.

**Comment by cbrnr** [comment]
Thanks, it works now - maybe I forgot to pull the latest changes or something. Sorry about the noise.

**Pull request merged**